platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/collections/AbstractLongList.java

```diff
@@ -143,16 +147,13 @@ protected AbstractLongList(final int numLongsPerChunk, final long maxLongs, fina
         }
         if (numLongsPerChunk > MAX_NUM_LONGS_PER_CHUNK) {
             throw new IllegalArgumentException(
-                    "Cannot store " + numLongsPerChunk + " per chunk (max is " + MAX_NUM_LONGS_PER_CHUNK + ")");
+                    String.format(CHUNK_SIZE_EXCEEDED_MSG, numLongsPerChunk, MAX_NUM_LONGS_PER_CHUNK));
```

Comment:
This reads better as `CHUNK_SIZE_EXCEEDED_MSG.formatted(numLongsPerChunk, MAX_NUM_LONGS_PER_CHUNK)`  with String.format really shouldn't be used much in Java 17 or later (`.formatted` is not available much before that)

Reply:
Interesting. Didn't know about it. Fixed.